test/test_plonk_component.hpp

```diff
@@ -178,10 +184,10 @@ namespace nil {
                 actor::zk::snark::placeholder_private_preprocessor<BlueprintFieldType, placeholder_params>::process(
                     bp, assignments.private_table(), desc, fri_params).get();
             auto proof = actor::zk::snark::placeholder_prover<BlueprintFieldType, placeholder_params>::process(
-                public_preprocessed_data, private_preprocessed_data, desc, bp, assignments, fri_params).get();
+                public_preprocessed_data, private_preprocessed_data, desc, bp, assignments, fri_params);
 
             bool verifier_res = actor::zk::snark::placeholder_verifier<BlueprintFieldType, placeholder_params>::process(
-                public_preprocessed_data, proof, bp, fri_params).get();
```

Comment:
Hmm is it true, that if we remove `.get()` - that means or `process` function now one thread only? Because in another case - we always try to return future if function inside or it's children inside have parallelism...

Reply:
This was a bug, this file is not used in many places. I ran a test, saw it, fixed it. Verifier's process function returns a bool, and operations are NOT over polynomials, but over FieldType::value_types. There's not parallelism inside. I believe verification of proof is so much faster that proof creation, that we did not care, at least for now.

Reply:
Oh, super! Thanks for watching!